packages/account/package.json

```diff
@@ -43,7 +43,7 @@
         "mobx-react": "^7.5.1",
         "onfido-sdk-ui": "^11.0.0",
         "prop-types": "^15.7.2",
-        "qrcode.react": "^1.0.0",
+        "qrcode.react": "^3.1.0",
```

Comment:
shouldnt we have changes to package-lock.json file?

Reply:
Also this qrcode.react is used in two-factor-authentication (https://app.deriv.com/account/two-factor-authentication). Can we add a sanity test for this in QA checklist

Reply:
sure 👍